src/adapter/graphql/tests/tests/test_gql_data.rs

```diff
@@ -66,6 +73,14 @@ async fn create_catalog_with_local_workspace(
             .add::<ObjectStoreRegistryImpl>()
             .add::<ObjectStoreBuilderLocalFs>()
             .add::<LoginPasswordAuthProvider>()
+            .add::<RebacServiceImpl>()
+            .add::<RebacServiceImplCacheState>()
+            .add::<InMemoryRebacRepository>()
+            .add_value(DefaultAccountProperties { is_admin: false })
+            .add_value(DefaultDatasetProperties {
+                allows_anonymous_read: false,
+                allows_public_read: false,
+            })
```

Comment:
Can we have these defined as shared constants somewhere? I think they repeated as well in a few suites, including the old one about datasets.